addons/mass_mailing/models/mailing.py

```diff
@@ -112,12 +112,18 @@ def _get_default_mail_server_id(self):
     reply_to = fields.Char(
         string='Reply To', compute='_compute_reply_to', readonly=False, store=True,
         help='Preferred Reply-To Address')
+    testing_mailing_id = fields.Many2one('mailing.ab.testing', string="Testing Mailing")
+    version_id = fields.Many2one('mailing.mailing.version', string="Version", copy=False)
```

Comment:
Adding a model to handle this feels really overkill.
Can't we simply have a char?
I checked to see why you need the color on the version model but I have not found any usage.

Problems with adding models like that is that it creates living records.
And those records, in this case are a bit technical, meaning there is no way to clean them.
We generally add debug views to be able to handle those (unlink unnecessary ones etc).
But here I feel like we could simplify instead?